### ISSUE TITLE
Way to temporarily disable?

### ISSUE THREAD

**Issue by chrismanderson** [jverdi/JVFloatLabeledTextField#151]
Dumb question but didn't see this functionality in the repo - any way to disable the floating behavior on a `JVFloatLabeledTextField` subclass? E.g. I'm looking for a boolean field to make the text field behave as normal, non `JVFloatLabeledTextField` field.

**Comment by jverdi** [comment]
No, this functionality doesn't exist today. If you work it out, I'd be happy to review a PR though.

**Comment by chrismanderson** [comment]
Sounds good, if I proceed with that functionality will submit as a PR for sure.
